src/ontology/oeo.omn

```diff
@@ -213,6 +213,12 @@ pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1136"
 ObjectProperty: OEO_00020182
 
     
+ObjectProperty: OEO_00020226
+
+    SubPropertyOf: 
+        <http://purl.obolibrary.org/obo/RO_0000057>
```

Comment:
Should this part be deleted from oeo.omn?

Reply:
Hm, definitely deleted entirely, but maybe moved to oeo-model or oeo-shared?

Reply:
I mean, it's redundant to oeo-model.

Reply:
Yes, you're right. Then delete it. I think, you can delete then the whole lines 216 to 221.